libs/template/config.go

```diff
@@ -70,6 +70,15 @@ func validateSchema(schema *jsonschema.Schema) error {
 
 // Reads json file at path and assigns values from the file
 func (c *config) assignValuesFromFile(path string) error {
+	// It's valid to set additional properties in the config file that are not
+	// defined in the schema. They are filtered below. For the duration of this
+	// function we disable the additional properties check, to allow those
+	// properties to be loaded.
+	c.schema.AdditionalProperties = true
+	defer func() {
```

Comment:
I'd rather move this right after .LoadInstance call to be more explicit

Reply:
done